src/main/webapp/app/exercises/shared/result/result.service.ts

```diff
@@ -20,11 +20,11 @@ export type EntityArrayResponseType = HttpResponse<Result[]>;
 
 export interface IResultService {
     find: (id: number) => Observable<EntityResponseType>;
-    findBySubmissionId: (submissionId: number) => Observable<EntityResponseType>;
+    findBySubmissionId: (participationId: number, submissionId: number) => Observable<EntityResponseType>;
```

Comment:
If I see this correctly, this method and the corresponding REST call on the server side are not used any more because they are outdated. Therefore please delete all occurrences and usages